app/javascript/crayons/CTAs/CTA.jsx

```diff
@@ -0,0 +1,22 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Link } from '@crayons';
+
+export const CTA = (props) => {
+  const { style = 'default', className, ...otherProps } = props;
```

Comment:
Refactor (blocking): Consider a prop name other than `style`, e.g. `variant`, as `style` is an out-of-the-box prop for inline styles in Preact.

Reply:
ohhh, of course.. silly me... fixing.